setup.py

```diff
@@ -0,0 +1,143 @@
+# The COPYRIGHT file at the top level of this repository contains
+# the full copyright notices and license terms.
+
+from setuptools import setup
+import re
+import os
+import io
+try:
+    from configparser import ConfigParser
+except ImportError:
+    from ConfigParser import ConfigParser
+
+MODULE = 'create_transactions'
+PREFIX = 'trytoncalidae'
+MODULE2PREFIX = {
+    'account_bank_statement': 'trytonspain',
+    'account_bank_statement_account': 'trytonspain',
+    'analytic_bank_statement': 'nantic',
+}
+
+
+def read(fname):
+    return io.open(
+        os.path.join(os.path.dirname(__file__), fname),
+        'r', encoding='utf-8').read()
+
+
+def get_require_version(name):
+    if minor_version % 2:
+        require = '%s >= %s.%s.dev0, < %s.%s'
+    else:
+        require = '%s >= %s.%s, < %s.%s'
+    require %= (
+        name, major_version, minor_version,
+        major_version, minor_version + 1
+    )
+    return require
+
+
+config = ConfigParser()
+config.readfp(open('tryton.cfg'))
+info = dict(config.items('tryton'))
+for key in ('depends', 'extras_depend', 'xml'):
+    if key in info:
+        info[key] = info[key].strip().splitlines()
+version = info.get('version', '0.0.1')
+major_version, minor_version, _ = version.split('.', 2)
+major_version = int(major_version)
+minor_version = int(minor_version)
+
+requires = [
+    'trytonspain_account_bank_statement @ '
+    'https://github.com/trytonspain/trytond-account_bank_statement/'
+    'archive/refs/heads/5.2.zip#'
+    'egg=trytonspain_account_bank_statement<5.3',
+    'trytonspain_account_bank_statement_account @ '
+    'https://github.com/trytonspain/trytond-account_bank_statement_account/'
+    'archive/refs/heads/5.2.zip#'
+    'egg=trytonspain_account_bank_statement_account<5.3',
+    'trytonspain_account_move_draft @ '
+    'https://github.com/trytonspain/trytond-account_move_draft/'
+    'archive/refs/heads/5.2.zip#'
+    'egg=trytonspain_account_move_draft<5.3',
+    'nantic_analytic_bank_statement @ '
+    'https://github.com/NaN-tic/trytond-analytic_bank_statement/'
+    'archive/refs/heads/5.2.zip#'
+    'egg=nantic_analytic_bank_statement<5.3',
+]
+
+for dep in info.get('depends', []):
+    if not re.match(r'(ir|res)(\W|$)', dep):
+        prefix = MODULE2PREFIX.get(dep, 'trytond')
+        requires.append(get_require_version('%s_%s' % (prefix, dep)))
+requires.append(get_require_version('trytond'))
+
+tests_require = []
+dependency_links = []
+
+if minor_version % 2:
+    # Add development index for testing with proteus
+    dependency_links.append('https://trydevpi.tryton.org/')
+
+setup(
+    name='%s_%s' % (PREFIX, MODULE),
+    version=version,
+    description='Create multiple transactions from bank statement line',
+    long_description=read('README.md'),
+    author='Calidae',
+    author_email='dev@calidae.com',
+    url='https://github.com/calidae/',
+    download_url="https://github.com/calidae/%s" % MODULE,
```

Comment:
This link
1) does not exist!
2) should point to the https://pypi.org/projects/trytoncalidae-xxxxx if the built package will be uploaded there. A github **archive** otherwise